gocourse05/go.mod

```
@@ -0,0 +1,3 @@
+module gocourse05
+
+go 1.22.6
```

Найновіша версія Go 1.23:

```suggestion
go 1.23.0
```

Тому прошу її використовувати. https://go.dev/dl/